core/src/test/java/org/bitcoinj/core/LegacyAddressTest.java

```diff
@@ -21,16 +21,14 @@
 import org.bitcoinj.params.Networks;
 import org.bitcoinj.params.TestNet3Params;
 import org.bitcoinj.script.Script;
+import org.bitcoinj.script.Script.ScriptType;
 import org.bitcoinj.script.ScriptBuilder;
 import org.bitcoinj.script.ScriptPattern;
-import org.bitcoinj.script.Script.ScriptType;
 import org.junit.Test;
 
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
+import java.io.*;
```

Comment:
I personally prefer individual imports, rather than the `*` syntax.

Reply:
Sorry my ide did it.